src/devices/schneider_electric.ts

```diff
@@ -494,6 +525,7 @@ const definitions: Definition[] = [
         fromZigbee: [fz.fan],
         toZigbee: [tzLocal.fan_mode],
         exposes: [e.fan().withModes(['off', 'low', 'medium', 'high', 'on'])],
+        extend: [fanIndicatorMode(), fanIndicatorOrientation()],
```

Comment:
I don't really understand how `extend` works, but I assume everything gets mapped correctly and you don't need to add any data/context into `fromZigbee` / `toZigbee`

Reply:
That's exactly how it works!